<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>959</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelTitle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>LINUX QT to STM32</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberTemperature">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelTemperature">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>현재 온도</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSendData">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSend">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>SEND</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditRxData">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>211</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelSendData">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Send Data</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelRxData">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Rx Data</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxLED1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>LED1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxLED2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>LED2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxLED3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>LED3</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="plotTemperature" native="true">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>481</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelHumidity">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>310</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>현재 습도</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberHumidity">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>370</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="plotHumidity" native="true">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>290</y>
      <width>481</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QDial" name="dialServo">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>130</y>
      <width>121</width>
      <height>111</height>
     </rect>
    </property>
    <property name="notchesVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDial" name="dialLED">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>430</y>
      <width>121</width>
      <height>111</height>
     </rect>
    </property>
    <property name="notchesVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelDialServo">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>220</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Servo</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDialLED">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>520</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>LED</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberServo">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>240</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberLED">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>540</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClear">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>All Clear</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSliderDHT11Interval">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>590</y>
      <width>331</width>
      <height>16</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="value">
     <number>99</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelDHT11Interval">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>570</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>DHT11 Interval</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberHorizontalSliderDHT11Interval">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>570</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="value" stdset="0">
     <double>99.000000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>959</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
